### DISCUSSION TITLE
An electron track bug

### DISCUSSION THREAD

**Discussion by ZhixinFan-hub** [epochpic/epoch#736] [Q&A]
During my attempt to track electron movement, I recorded the electron's speed (v), position (r), and the corresponding time (t). The speed, v, consistently remained below the speed of light, c. However, when calculating the velocity using discrete differences in Matlab, specifically with the formula ((r(i) - r(i-1)) / (t(i) - t(i-1))), the result occasionally exceeded c slightly, reaching up to about 1.000088884938731c in one of the simulations.

This discrepancy could potentially impact calculations related to electron radiation, especially when determining the retarded time for each position.

**Comment by Status-Mirror** [comment]
Hey @ZhixinFan-hub,

How were you calculating the trajectory of a specific particle? Can you paste the input deck?

Cheers,
Stuart
